<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1029</width>
    <height>765</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="preview_label">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>711</width>
      <height>711</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="alignment">
     <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignTop</set>
    </property>
   </widget>
   <widget class="QWidget" name="verticalLayoutWidget">
    <property name="geometry">
     <rect>
      <x>713</x>
      <y>10</y>
      <width>301</width>
      <height>421</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <widget class="QTabWidget" name="operations_widget">
       <property name="currentIndex">
        <number>3</number>
       </property>
       <widget class="QWidget" name="tab">
        <attribute name="title">
         <string>Closest neighbour</string>
        </attribute>
        <widget class="QWidget" name="gridLayoutWidget">
         <property name="geometry">
          <rect>
           <x>0</x>
           <y>10</y>
           <width>291</width>
           <height>211</height>
          </rect>
         </property>
         <layout class="QGridLayout" name="gridLayout">
          <item row="0" column="1">
           <spacer name="horizontalSpacer_2">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>40</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
          <item row="0" column="0">
           <widget class="QPushButton" name="closest_neighbor_use_button">
            <property name="text">
             <string>Use</string>
            </property>
           </widget>
          </item>
          <item row="2" column="0">
           <spacer name="verticalSpacer">
            <property name="orientation">
             <enum>Qt::Vertical</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>20</width>
              <height>40</height>
             </size>
            </property>
           </spacer>
          </item>
          <item row="1" column="0">
           <widget class="QDoubleSpinBox" name="doubleSpinBox">
            <property name="keyboardTracking">
             <bool>false</bool>
            </property>
            <property name="minimum">
             <double>0.050000000000000</double>
            </property>
            <property name="maximum">
             <double>100.000000000000000</double>
            </property>
            <property name="singleStep">
             <double>0.050000000000000</double>
            </property>
            <property name="value">
             <double>1.000000000000000</double>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </widget>
       <widget class="QWidget" name="tab_2">
        <attribute name="title">
         <string>Bilinear interpolation</string>
        </attribute>
        <widget class="QWidget" name="gridLayoutWidget_2">
         <property name="geometry">
          <rect>
           <x>0</x>
           <y>10</y>
           <width>291</width>
           <height>211</height>
          </rect>
         </property>
         <layout class="QGridLayout" name="gridLayout_2">
          <item row="0" column="0">
           <widget class="QPushButton" name="bilinear_use_button">
            <property name="text">
             <string>Use</string>
            </property>
           </widget>
          </item>
          <item row="1" column="0">
           <widget class="QDoubleSpinBox" name="bilinear_spinbox">
            <property name="minimum">
             <double>0.050000000000000</double>
            </property>
            <property name="maximum">
             <double>100.000000000000000</double>
            </property>
            <property name="singleStep">
             <double>0.050000000000000</double>
            </property>
            <property name="value">
             <double>1.000000000000000</double>
            </property>
           </widget>
          </item>
          <item row="2" column="0">
           <spacer name="verticalSpacer_2">
            <property name="orientation">
             <enum>Qt::Vertical</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>20</width>
              <height>40</height>
             </size>
            </property>
           </spacer>
          </item>
          <item row="0" column="1">
           <spacer name="horizontalSpacer_3">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>40</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
         </layout>
        </widget>
       </widget>
       <widget class="QWidget" name="tab_3">
        <attribute name="title">
         <string>Biqubic interpolation</string>
        </attribute>
        <widget class="QWidget" name="gridLayoutWidget_4">
         <property name="geometry">
          <rect>
           <x>0</x>
           <y>10</y>
           <width>291</width>
           <height>211</height>
          </rect>
         </property>
         <layout class="QGridLayout" name="gridLayout_4">
          <item row="2" column="0">
           <spacer name="verticalSpacer_3">
            <property name="orientation">
             <enum>Qt::Vertical</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>20</width>
              <height>40</height>
             </size>
            </property>
           </spacer>
          </item>
          <item row="0" column="0">
           <widget class="QPushButton" name="biqubic_use_button">
            <property name="text">
             <string>Use</string>
            </property>
           </widget>
          </item>
          <item row="1" column="0">
           <widget class="QDoubleSpinBox" name="biqubic_spinbox">
            <property name="minimum">
             <double>0.050000000000000</double>
            </property>
            <property name="maximum">
             <double>100.000000000000000</double>
            </property>
            <property name="singleStep">
             <double>0.050000000000000</double>
            </property>
            <property name="value">
             <double>1.000000000000000</double>
            </property>
           </widget>
          </item>
          <item row="0" column="1">
           <spacer name="horizontalSpacer_4">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>40</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
         </layout>
        </widget>
       </widget>
       <widget class="QWidget" name="tab_4">
        <attribute name="title">
         <string>Rotator</string>
        </attribute>
        <widget class="QWidget" name="gridLayoutWidget_5">
         <property name="geometry">
          <rect>
           <x>0</x>
           <y>10</y>
           <width>291</width>
           <height>211</height>
          </rect>
         </property>
         <layout class="QGridLayout" name="gridLayout_5">
          <item row="0" column="1">
           <widget class="QPushButton" name="rotator_use_button">
            <property name="text">
             <string>Use</string>
            </property>
           </widget>
          </item>
          <item row="2" column="1">
           <widget class="QLCDNumber" name="rotator_angle_lcd">
            <property name="value" stdset="0">
             <double>0.000000000000000</double>
            </property>
           </widget>
          </item>
          <item row="3" column="1">
           <spacer name="verticalSpacer_4">
            <property name="orientation">
             <enum>Qt::Vertical</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>20</width>
              <height>40</height>
             </size>
            </property>
           </spacer>
          </item>
          <item row="1" column="1">
           <widget class="QPushButton" name="rotator_center_button">
            <property name="text">
             <string>Choose center</string>
            </property>
           </widget>
          </item>
          <item row="2" column="2">
           <widget class="QSlider" name="rotator_angle_slider">
            <property name="minimum">
             <number>-180</number>
            </property>
            <property name="maximum">
             <number>180</number>
            </property>
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
           </widget>
          </item>
          <item row="1" column="2">
           <layout class="QHBoxLayout" name="horizontalLayout_2">
            <item>
             <widget class="QLCDNumber" name="rotator_center_x">
              <property name="value" stdset="0">
               <double>-1.000000000000000</double>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLCDNumber" name="rotator_center_y">
              <property name="value" stdset="0">
               <double>-1.000000000000000</double>
              </property>
             </widget>
            </item>
           </layout>
          </item>
         </layout>
        </widget>
       </widget>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>720</x>
      <y>260</y>
      <width>291</width>
      <height>101</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <layout class="QVBoxLayout" name="verticalLayout_2">
       <item>
        <widget class="QPushButton" name="load_image_button">
         <property name="text">
          <string>Load Image</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="save_image_button">
         <property name="text">
          <string>Save Image</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QCheckBox" name="unlock_cursor_checkbox">
         <property name="text">
          <string>Unlock cursor</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QVBoxLayout" name="verticalLayout_3">
       <item>
        <widget class="QPushButton" name="reset_button">
         <property name="text">
          <string>Reset</string>
         </property>
        </widget>
       </item>
       <item>
        <layout class="QGridLayout" name="gridLayout_3">
         <item row="0" column="0">
          <widget class="QLCDNumber" name="x1_lcd"/>
         </item>
         <item row="0" column="1">
          <widget class="QLCDNumber" name="y1_lcd"/>
         </item>
         <item row="1" column="1">
          <widget class="QLCDNumber" name="y2_lcd"/>
         </item>
         <item row="1" column="0">
          <widget class="QLCDNumber" name="x2_lcd"/>
         </item>
        </layout>
       </item>
      </layout>
     </item>
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1029</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
